server/routes/offenceCodeDecisions/anotherDecisionHelper.ts

```diff
@@ -0,0 +1,44 @@
+// All functionality that knows about the prison decision.
```

Comment:
The name - another?

Reply:
Yeah - rubbish name - can go with anotherPerson?

Reply:
for this PR otherPerson - after change all to victim....